rails/Dockerfile

```diff
@@ -8,7 +8,7 @@
 # == Development Image
 #
 # Also used as the base for building assets for prod images.
-FROM ruby:2.7-alpine as devcore
```

Comment:
Should I close https://github.com/Terrastories/terrastories/pull/910? Or should I update that PR to be for updating from 2.7.7 to 2.7.8?

Reply:
Let's keep it and have it update from 2.7.7 to 2.7.8!